addons/pulsar/templates/componentdef-zookeeper.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ComponentDefinition
+metadata:
+  name: pulsar-zookeeper
+  labels:
+      {{- include "pulsar.labels" . | nindent 4 }}
+spec:
+  provider: kubeblocks
+  description: Pulsar zookeeper.
+  serviceKind: pulsar
+  serviceVersion: {{ default .Chart.AppVersion .Values.clusterVersionOverride }}
+  updateStrategy: BestEffortParallel
```

Comment:
Serial would be better

Reply:
for components without kb role detection logic, RSM currently has bugs under the SerialUpdateStrategy strategy(issue: https://github.com/apecloud/kubeblocks/issues/6785). so try the BestEffortParallel strategy to ensure normal function.